design/001-design-crd-v1beta1.md

```diff
@@ -75,6 +86,7 @@ spec:
   target:
     name: my-secret
     creationPolicy: 'Merge'
+    deletionPolicy: 'None' #Possible values are None, Merge, Delete - TBC during implementation.
```

Comment:
I get `None` and `Delete` - but what could merge mean :thinking:?

Reply:
For GetAllSecrets: if one secret in the provider doesn’t exist anymore, merge would just remove any secret keys not available anymore. This can  also be useful if creationPolicy was merge itself.

Merge would be “remove keys in the secret, but not the secret”

Reply:
I see, that makes sense.